scripts/snap-pac

```diff
@@ -20,6 +20,8 @@
 
 set -o errtrace
 
+[[ ! -z "$SNAP_PAC_IGNORE" ]] && { echo "snap-pac: skipping..."; exit; }
```

Comment:
Instead of ! -z und -n
I suggest to specify the exit code, in this case exit 0
If you print "skippin" I'd also name the option SNAP_PAC_SKIP
Does the code work if the variable i unset?

Reply:
Done x3, thanks 🙂

When I run `$ SNAP_PAC_SKIP=true sudo pacman -S ...`, I see `skipping` two times, when I run `$ sudo pacman -S ...` I see IDs of created snapshots, so seems to work as expected.